### ISSUE TITLE
Implement Service-Worker

### ISSUE THREAD

**Issue by pierman1** [rijkvanzanten/obachine#166]

**Comment by pierman1** [comment]
Copy pasta!

**Comment by Mimaaa** [comment]
@dandevri 🙅‍♂️

**Comment by Mimaaa** [comment]
I'm gonna tackle this one with the [Service-Worker Webpack Plugin](https://github.com/oliviertassinari/serviceworker-webpack-plugin)

**Comment by dandevri** [comment]
Such wow. 🦄 The webpack plugin only makes registering the SW easier. You still need to write you own `sw.js`.  See step 3 in the `readme.md`.

**Comment by Mimaaa** [comment]
True, that's where you come in. 🙅‍♂️ But I'll probably go with [this one](https://github.com/NekR/offline-plugin) as it has [AppCache](https://caniuse.com/#search=appcache) as a fallback, thus supporting a lot more browsers if SW support fails.
